pjmedia/src/pjmedia/sdp_neg.c

```diff
@@ -183,6 +211,9 @@ PJ_DEF(pj_status_t) pjmedia_sdp_neg_create_w_remote_offer(pj_pool_t *pool,
      * SDPs pool after a successful SDP nego.
      */
     neg->pool_active = pool;
+#if PJMEDIA_SDP_NEG_MAINTAIN_SESSION_PT
+    neg->pool = pool;
```

Comment:
be careful that the pool is a flip-flop one.

Reply:
The call to `pjmedia_sdp_neg_create_w_remote_offer()` with temporary pool is called to verify the incoming request `pjsip_inv_verify_request3()`. So it's not used long term, it should be safe